main.tf

```diff
@@ -0,0 +1,170 @@
+}
+
+resource "aws_security_group_rule" "ingress_service" {
+  security_group_id = "${aws_security_group.ecs_service.id}"
+  type              = "ingress"
+  protocol          = "icmp"
+  from_port         = "8"
+  to_port           = "0"
+  cidr_blocks       = ["0.0.0.0/0"]
+}
+
+resource "aws_security_group_rule" "egress_service" {
+  security_group_id = "${aws_security_group.ecs_service.id}"
+  type              = "egress"
+  protocol          = "-1"
+  from_port         = 0
+  to_port           = 0
+  cidr_blocks       = ["0.0.0.0/0"]
+}
+
+# ------------------------------------------------------------------------------
+# LB Target group
+# ------------------------------------------------------------------------------
+resource "aws_lb_target_group" "task" {
+  vpc_id       = "${var.vpc_id}"
+  protocol     = "${var.task_container_protocol}"
+  port         = "${var.task_container_port}"
+  target_type  = "ip"
+  health_check = ["${var.health_check}"]
+
+  # NOTE: TF is unable to destroy a target group while a listener is attached,
+  # therefor we have to create a new one before destroying the old. This also means
+  # we have to let it have a random name, and then tag it with the desired name.
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  tags = "${merge(var.tags, map("Name", "${var.name_prefix}-target-${var.task_container_port}"))}"
+}
+
+# ------------------------------------------------------------------------------
+# ECS Task/Service
+# ------------------------------------------------------------------------------
+data "null_data_source" "task_environment" {
+  count = "${var.task_definition_environment_count}"
+
+  inputs = {
+    name  = "${element(keys(var.task_definition_environment), count.index)}"
+    value = "${element(values(var.task_definition_environment), count.index)}"
+  }
+}
+
+resource "aws_ecs_task_definition" "task" {
+  family                   = "${var.name_prefix}"
+  execution_role_arn       = "${aws_iam_role.execution.arn}"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "${var.task_definition_cpu}"
+  memory                   = "${var.task_definition_memory}"
+  task_role_arn            = "${aws_iam_role.task.arn}"
+
+  container_definitions = <<EOF
+[{
+    "cpu":0,
+    "name": "${var.name_prefix}",
+    "image": "${var.task_definition_image}",
+    "essential": true,
+    "portMappings": [
+        {
+            "containerPort": ${var.task_container_port},
+            "hostPort": ${var.task_container_port},
+            "protocol":"tcp"
+        }
+    ],
+    "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+            "awslogs-group": "${aws_cloudwatch_log_group.main.name}",
+            "awslogs-region": "${data.aws_region.current.name}",
+            "awslogs-stream-prefix": "container"
+        }
+    },
+    "environment": ${jsonencode(data.null_data_source.task_environment.*.outputs)}
```

Comment:
Might be a good idea to export the command while we are at it, so this is on par with the `ecs/service` module:
https://github.com/telia-oss/terraform-aws-ecs/blob/master/modules/service/main.tf#L88

Reply:
Sure thing 👍